#! /bin/sh /usr/share/dpatch/dpatch-run
## dbug336890-kFreeBSD.dpatch by  Aurelien Jarno <aurel32@debian.org>; 
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: http://bugs.debian.org/336890

@DPATCH@
diff -urNad portaudio-18.1~/pa_unix_oss/pa_unix_oss.c portaudio-18.1/pa_unix_oss/pa_unix_oss.c
--- portaudio-18.1~/pa_unix_oss/pa_unix_oss.c	2003-06-30 12:05:50.000000000 +0100
+++ portaudio-18.1/pa_unix_oss/pa_unix_oss.c	2007-03-17 12:12:09.000000000 +0000
@@ -44,7 +44,7 @@
 #ifdef __linux__
 #include <linux/soundcard.h>
 #else
-#include <machine/soundcard.h> /* JH20010905 */
+#include <sys/soundcard.h> /* JH20010905 */
 #endif
 
 
